test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BasicAuthTests.cs

```diff
@@ -240,5 +240,54 @@ public async Task GivenAUserWithConvertDataPermissions_WhenConvertData_TheServer
             var result = await tempClient.ConvertDataAsync(parameters);
             Assert.NotEmpty(result);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenUserWithNoProfileAdminPermission_WhenCreate_ThenServerShouldReturnForbidden()
```

Comment:
Should the test name reference that the user is trying to create (or other CUD action) a resource type used by profile? Right now it reads as if a user needs profile admin permission to perform CUD actions on any resource type.